app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :user
+  belongs_to :user
```

Comment:
- [ ] I think you repeated the relation here since you wrote it above but I think you mean relation to post so kindly update to be belongs_to :post 🔥😊